From 84f94861da7caf5e5ceb73c52cb11911ffebbad5 Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Fri, 15 Sep 2017 15:37:22 +0800
Subject: [PATCH 2541/5242] MLK-16471-02 arm64: dts: imx8qm/imx8qxp: remove
 ptp clock from assigned-clocks

commit  7fb81ccda056b1d3b38c7ace4457230a2ba6b506 from
https://source.codeaurora.org/external/imx/linux-imx.git

Since IMX8QXP_ENET0_ROOT_DIV clock is included in assigned-clocks list,
and who is the parent of IMX8QXP_ENET0_PTP_CLK (ptp clock), so remove
ptp clock from assigned-clocks list.

(BuildInfo: SCFW 3e70523d, IMX-MKIMAGE 0, ATF 0)

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Reviewed-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi  |   10 ++++------
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |   10 ++++------
 2 files changed, 8 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 68ef44a..78b9d34 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -2251,9 +2251,8 @@
 			<&clk IMX8QM_ENET0_PTP_CLK>, <&clk IMX8QM_ENET0_TX_CLK>;
 		clock-names = "ipg", "ahb", "enet_clk_ref", "ptp", "enet_2x_txclk";
 		assigned-clocks = <&clk IMX8QM_ENET0_ROOT_DIV>,
-				  <&clk IMX8QM_ENET0_REF_DIV>,
-				  <&clk IMX8QM_ENET0_PTP_CLK>;
-		assigned-clock-rates = <250000000>, <125000000>, <125000000>;
+				  <&clk IMX8QM_ENET0_REF_DIV>;
+		assigned-clock-rates = <250000000>, <125000000>;
 		fsl,num-tx-queues=<3>;
 		fsl,num-rx-queues=<3>;
 		power-domains = <&pd_conn_enet0>;
@@ -2272,9 +2271,8 @@
 			<&clk IMX8QM_ENET1_PTP_CLK>, <&clk IMX8QM_ENET1_TX_CLK>;
 		clock-names = "ipg", "ahb", "enet_clk_ref", "ptp", "enet_2x_txclk";
 		assigned-clocks = <&clk IMX8QM_ENET1_ROOT_DIV>,
-				  <&clk IMX8QM_ENET1_REF_DIV>,
-				  <&clk IMX8QM_ENET1_PTP_CLK>;
-		assigned-clock-rates = <250000000>, <125000000>, <125000000>;
+				  <&clk IMX8QM_ENET1_REF_DIV>;
+		assigned-clock-rates = <250000000>, <125000000>;
 		fsl,num-tx-queues=<3>;
 		fsl,num-rx-queues=<3>;
 		power-domains = <&pd_conn_enet1>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index df57a57..2a7aa6d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1922,9 +1922,8 @@
 			<&clk IMX8QXP_ENET0_PTP_CLK>, <&clk IMX8QXP_ENET0_TX_CLK>;
 		clock-names = "ipg", "ahb", "enet_clk_ref", "ptp", "enet_2x_txclk";
 		assigned-clocks = <&clk IMX8QXP_ENET0_ROOT_DIV>,
-				  <&clk IMX8QXP_ENET0_REF_DIV>,
-				  <&clk IMX8QXP_ENET0_PTP_CLK>;
-		assigned-clock-rates = <250000000>, <125000000>, <125000000>;
+				  <&clk IMX8QXP_ENET0_REF_DIV>;
+		assigned-clock-rates = <250000000>, <125000000>;
 		fsl,num-tx-queues=<3>;
 		fsl,num-rx-queues=<3>;
 		power-domains = <&pd_conn_enet0>;
@@ -1942,9 +1941,8 @@
 			<&clk IMX8QXP_ENET1_PTP_CLK>, <&clk IMX8QXP_ENET1_TX_CLK>;
 		clock-names = "ipg", "ahb", "enet_clk_ref", "ptp", "enet_2x_txclk";
 		assigned-clocks = <&clk IMX8QXP_ENET1_ROOT_DIV>,
-				  <&clk IMX8QXP_ENET1_REF_DIV>,
-				  <&clk IMX8QXP_ENET1_PTP_CLK>;
-		assigned-clock-rates = <250000000>, <125000000>, <125000000>;
+				  <&clk IMX8QXP_ENET1_REF_DIV>;
+		assigned-clock-rates = <250000000>, <125000000>;
 		fsl,num-tx-queues=<3>;
 		fsl,num-rx-queues=<3>;
 		power-domains = <&pd_conn_enet1>;
-- 
1.7.9.5

